### ISSUE TITLE
Adding an application to Konflux - better guidance for the user? 

### ISSUE THREAD

**Issue by shairevivo** [konflux-ci/konflux-ci#1017]
When trying to select pipeline option, it is difficult to understand the diffs between the options.
Either we provide more "friendly names) or at least some explanation? 
![2024-12-17_11-13](https://github.com/user-attachments/assets/ff9f3875-0328-4336-9a88-1b51a4f83a26)

**Comment by yftacherzog** [comment]
I agree.

Do you think the [documentation here](https://konflux-ci.dev/docs/advanced-how-tos/installing/enabling-builds/#docker-build-oci-ta) will be useful?

If so, we can as immediate step, add this link to our README in the relevant location, and then also open a ticket to the UI team to see if we can introduce a similar link or explanation on the UI itself.

Let us know your thoughts.

**Comment by shairevivo** [comment]
Yes, link to documentation can be a good first step here. longer term we should think about different naming.
BTW, if you look at the first sentence in each one there is a good option for a name.
